.github/CODEOWNERS

```diff
@@ -4,7 +4,7 @@
 
 /runtime/ @boydc2014 @luhan2017 @carlosscastro @benbrown
 
-/Composer/ @cwhitten @boydc2014 @a-b-r-o-w-n @beyackle @srinaath @tonyanziano @geoffcoxmsft @hatpick
+/Composer/ @cwhitten @boydc2014 @a-b-r-o-w-n @beyackle @srinaath @tonyanziano @geoffcoxmsft @hatpick @benbrown @pavolum
```

Comment:
can you have a dedicated PR for this? PRs are good because we can revert them if they cause an unwanted issue, this part of the PR has nothing to do with the rest of it.

Reply:
Sure thing!